client/src/components/HostViews/Waiting.vue

```diff
@@ -86,6 +127,25 @@ export default {
     },
     qrCodeAPI() {
       return `https://api.qrserver.com/v1/create-qr-code/?size=150x150&data=${this.url}`
+    },
+    roomId() {
+      return this.$store.state.roomid
+    },
+    displayUrl() {
+      return `${window.location.href.substring(0, window.location.href.length - 5)}/join`
```

Comment:
A comment with an example url would be nice.

Reply:
Good idea, will add